plugins/draco/module/vtkF3DDracoReader.cxx

```diff
@@ -14,8 +14,12 @@
 #include <vtkPolyData.h>
 
 #include "draco/compression/decode.h"
+#include "draco/draco_features.h"
 #include "draco/io/stdio_file_reader.h"
 
+#ifndef DRACO_MESH_COMPRESSION_SUPPORTED
+#error "Please rebuild draco with DRACO_MESH_COMPRESSION cmake option enabled."
```

Comment:
Can we detect that at configuration time by checking a cmake variable?

Reply:
nope, thanks draco